src/main/java/mil/dds/anet/resources/AnetEmailResource.java

```diff
@@ -0,0 +1,26 @@
+package mil.dds.anet.resources;
+
+import static mil.dds.anet.utils.AuthUtils.assertAdministrator;
+import static mil.dds.anet.utils.DaoUtils.getUserFromContext;
+
+import io.leangen.graphql.annotations.GraphQLQuery;
+import io.leangen.graphql.annotations.GraphQLRootContext;
+import java.util.List;
+import java.util.Map;
+import mil.dds.anet.AnetObjectEngine;
+import mil.dds.anet.beans.AnetEmail;
+import mil.dds.anet.database.EmailDao;
+
+public class AnetEmailResource {
+  private final EmailDao emailDao;
+
+  public AnetEmailResource(final AnetObjectEngine engine) {
+    this.emailDao = engine.getEmailDao();
+  }
+
+  @GraphQLQuery(name = "pendingEmails")
+  public List<AnetEmail> getPendingEmails(@GraphQLRootContext Map<String, Object> context) {
+    assertAdministrator(getUserFromContext(context));
```

Comment:
To keep this in line with all other code, please import the classes `AuthUtils` and `DaoUtils`, and not the methods.